app/routes/data-quality.ts

```diff
@@ -1,3 +1,191 @@
+              },
+            },
+            sessions: {
+              'governing-body': {
+                'administrative-unit': {
+                  location: {
+                    label: params.municipalityLabel || undefined,
+                  },
+                },
+              },
+            },
+          },
+          size: 1,
+        });
+
+      // get all classifications (governing bodies) from the store and dont get duplicates
+      const allGoverningBodyClassifications = await this.store.findAll(
+        'governing-body-classification-code'
+      );
+
+      // put all of the classifications through a loop to get the count of agenda items per classification
+
+      const agendaItemsPromises = allGoverningBodyClassifications
+        .filter(
+          // filter out duplicates
+          (classification, index, self) =>
+            self.findIndex((c) => c.label === classification.label) === index
+        )
+        .map(async (governingBody) => {
+          const agendaItems: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+            await this.store.query('agenda-item', {
+              filter: {
+                sessions: {
+                  'governing-body': {
+                    'administrative-unit': {
+                      location: {
+                        label: params.municipalityLabel || undefined,
+                      },
+                    },
+                    classification: {
+                      label: governingBody.label,
+                    },
+                  },
+                },
+              },
+              size: 1,
+            });
+
+          return {
+            classification: governingBody.label,
+            count: getCount(agendaItems) || 0,
+          };
+        });
+
+      // resolve the promises
+      agendaItemsPerGoverningBodyClassification = await Promise.all(
+        agendaItemsPromises
+      );
+
+      agendaItemsPerGoverningBodyClassification =
+        agendaItemsPerGoverningBodyClassification.map((item) => ({
+          ...item,
+          percentage:
+            ((item.count || 0) / (getCount(totalCountAgendaItems) || 1)) * 50,
+        }));
+
+      agendaItemsPerGoverningBodyClassification.push({
+        classification: 'Totaal',
+        count: getCount(totalCountAgendaItems) || 0,
+        percentage: 50,
+      });
+
+      // get the total count of agenda items which have a vote or voters
+
+      const agendaItemsWithVoteOrVoters: AdapterPopulatedRecordArrayWithMeta<VoteModel> =
+        await this.store.query('vote', {
+          'filter[:or:][handled-by][subject][sessions][governing-body][is-time-specialization-of][administrative-unit][location][label]':
+            params.municipalityLabel || undefined,
+          'filter[:or:][handled-by][subject][sessions][governing-body][administrative-unit][location][label]':
+            params.municipalityLabel || undefined,
+          size: 1,
+        });
+
+      return {
```

Comment:
**Nitpick:** In order to improve the readability. Can you keep the return object simple and execute all the computations before?